src/MigrationLog/Migrations/CreateMigrationsTable.php

```diff
@@ -43,11 +43,17 @@ public static function timestamp()
         return strtotime('1970-01-01 00:00');
     }
 
+    /**
+     * @unreleased Add Check whether table installed before adding it to database.
+     * @throws DatabaseMigrationException
+     */
     public function run()
     {
-        global $wpdb;
+        if( $this->isTableExist() ) {
```

Comment:
@ravinderk instead of adding a bespoke check, would it work to update the run query with `CREATE TABLE IF NOT EXISTS`?

Reply:
fixed: https://github.com/impress-org/givewp/compare/9623a43e1f3e...4bc205debe92